content/_index.md

```diff
@@ -18,9 +18,17 @@ Contrary to the name's appearance, GitBOM is neither [git](/glossary/git) nor an
 {{< /home/section >}}
 
 {{< home/section title="Why?" >}}
-By correlating every software [artifact](/glossary/artifact) to a verifiable, complete, and concise [artifact tree](/glossary/artifact_tree), GitBOM enables run-time detection of potential vulnerabilities, regardless of the depth in a depencency tree from which that vulnerability originated.
+By constructing a complete, concise, and verifiable [artifact tree](/glossary/artifact_tree) for every [software artifact](/glossary/artifact), [GitBOM](/glossary/gitbom) enables:
+- Run-time detection of potential vulnerabilities, regardless of the depth in a dependency tree from which that vulnerability originated
+- Identifying license obligations across both open source and commercial software
+- More reliable attestation
+- Post-exploit forensics
 
-In short, it would let anyone easily answer the question, "does this product contain log4j?".
+By creating a unique, immutable, verifiable identifier (the [GitBOM ID](/glossary/gitbom/#gitbom-identifier)) for every [software artifact](/glossary/artifact), [GitBOM](/glossary/gitbom):
+- enables any [software artifact](/glossary/artifact) to be definitively linked to any metadata ([SBOM](/glossary/sbom), license, support info, security advisories, etc.)
```

Comment:
Not really? The SBOM can reference a GitBOM, but not the other way around -- GitBOM does not "link to" the metadata about the software package.

I often speak about how the current EO Min Req's for software identification are insufficient for certain use cases, but I wouldn't go so far as to say this "definitively links to metadata".

Reply:
Yes, the order in which this is stated matters. To be more precise, this is what was intended:
```suggestion
- enables any metadata ([SBOM](/glossary/sbom), license, support info, security advisories, etc.) to be linked to a specific set of corresponding [software artifacts](/glossary/artifact)
```

Reply:
Does this address your concern?

Reply:
Yep, that parses more clearly to me. Thanks!